src/etcd/client.py

```diff
@@ -9,12 +9,12 @@
 import logging
 #from http.client import HTTPException
 from aiohttp.web_exceptions import HTTPException
-try:
```

Comment:
Why do you remove this try/except?

Reply:
Because in my version of aiohttp both exceptions seemed to be present which made the except not used while the payload error is in the client error.